src/assets/css/bootstrap.min.css

```diff
@@ -0,0 +1 @@
+@import url(https://maxcdn.bootstrapcdn.com/bootstrap/3.4.1/css/bootstrap.min.css);
```

Comment:
Copy this url. 
Paste it in a browser 
Copy content of what will be displayed there
Paste it in this file instead of this import